app/assets/javascripts/3dbio_viewer/src/webapp/components/network/Label.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import i18n from "../../utils/i18n";
+import "./Network.css";
+
+interface LabelProps {
+    forText: string;
+    labelText: string;
```

Comment:
[minor] Label is implicitly a text, and in TS we have types, so usually we'd write just `label: string`.